### DISCUSSION TITLE
Next Release Date

### DISCUSSION THREAD

**Discussion by lluchini** [git-lfs/git-lfs#5098] [General] [answered]
Hello,

I would like to know when is planned the next release. My team is really in need of fix #5066.

Is there a calendar somewhere, or does someone has any idea when?

Thanks.

**Comment by bk2204** [comment, marked as answer]
Hey,

We have a few things to work on in the v3.3.0 milestone, and the core team has been busy elsewhere.  My conjecture is that it will likely end up being somewhere between mid-September and early October.

You are of course free to build from the latest main branch if you need features sooner.

**Reply by lluchini** [reply]
Thanks @bk2204 , I built it and it seems to solve our afflictions for the time being... :)

I'm not very big on open source contributions to the open source community (mostly because of time and fatigue), but as this tool is very important to the company, perhaps I'll be able to contribute on this one...

As the possibility of the above became reality, (and some open issues worry me) I would like to know more about the go (never used before), and how to start and debug the git-lfs. I'm not asking you to mentor me (sure it's welcome, but don't want to abuse it), just point some north to start helping here.

Anyway, thank you for your time.

**Reply by srohmen** [reply]
just out of curiosity (and of course I like a LFS version released which contains the patch I contributed ;) ), do all tickets of the milestone need to be solved before the release will be published?
https://github.com/git-lfs/git-lfs/milestone/34

**Reply by bk2204** [reply]
We're planning to get through the vast majority of them, if not all.  The proposals may or may not be implemented, but a lot of the rest of them are bugs, and we'd like to fix those.
